src/components/App/App.js

```diff
@@ -9,77 +9,59 @@ import scrollImg from "../../assets/scroll.png";
-    //    "recipients": [rec1, rec2, rec3],
-    //    "executors": [exc1, exc2, exc3],
-    //     "videos": [v1, v2, v3]
-    //     } 
-  })
+  const [user, setUser] = useState({})
   const [error, setError] = useState("")
 
-  // const getUser = async () => {
-  //   const url = 'https://elegy-backend.herokuapp.com/api/v1/user/1'
-  //   setError('')
-
-  //   try {
-  //     const response = await fetch(url)
-  //     const userData = await response.json()
-  //     console.log(userData)
-  //     setUser(userData.data)
-  //   } catch (err) {
-  //     setError(err.message)
-  //   }
-  // }
+  const getUser = async () => {
+    const url = 'https://elegy-backend.herokuapp.com/api/v1/user/1'
+    setError('')
+    try {
+      const response = await fetch(url)
+      const userData = await response.json()
+      console.log(userData)
+      setUser(userData.data)
+    } catch (err) {
+      setError(err.message)
+    }
+  }
 
-  // useEffect(() => {
-  //   getUser()
-  // }, [])
+  useEffect(() => {
+    getUser()
+    console.log(user)
+  }, [])
 
   return (
     <main>
       <Link to="/" style={{textDecoration: 'none'}}><Header /></Link>
-      <Switch>
-        <Route exact from='/'>
-          <section className='window'>
-            <NavLink className='preview-pane' exact to='/preview'><img src={preview}/></NavLink>
-            <NavLink className='countdown-pane' exact to='/countdown'><img src={sundial}/></NavLink>
-            <NavLink className='executor-pane' exact to='/executors'><img src={suit}/></NavLink>
-            <div className='placeholder-1'>
-              Placeholder
-            </div>
-            <NavLink className='obit-pane' exact to='/obituary'><img src={scrollImg}/></NavLink>
-            <NavLink className='timeline-pane' exact to='/timeline'><img src={timelineImg}/></NavLink>
-            <NavLink className='recipient-pane' exact to='/recipients'><img src={familyTree}/></NavLink>
-            <NavLink className='gallery-pane' exact to='/gallery'><img src={galleryImg}/></NavLink>
-            <div className='placeholder-2'>
-              Placeholder
-            </div>
-          </section>
-        </Route>
-        <Route exact path="/countdown" render={() => <CountdownPane etd={user.attributes.etd} err={error} dob={user.attributes.date_of_birth}/>}/>
-        <Route exact path="/obituary" render={() => <ObitPane />}/>
-      </Switch>
+      {!user.attributes 
+      ? <p className='loading-message'>Loading...</p>
+      : <Switch>
+          <Route exact from='/'>
+            <section className='window'>
+              <NavLink className='preview-pane' exact to='/preview'><img src={preview}/></NavLink>
+              <NavLink className='countdown-pane' exact to='/countdown'><img src={sundial}/></NavLink>
+              <NavLink className='executor-pane' exact to='/executors'><img src={suit}/></NavLink>
+              <div className='placeholder-1'>
+                Placeholder
```

Comment:
Do we still need these placeholders?

Reply:
Yes they are those triangles on the sides

Reply:
Oops you're right we don't need that text between the opening and closing tags